lib/bootloader_setup.pm

```diff
@@ -505,6 +505,10 @@ sub bootmenu_default_params {
         }
 
     }
+    if (get_var('AGAMA_AUTO')) {
+        my $path = data_url(get_var('AGAMA_AUTO'));
+        set_var('EXTRABOOTPARAMS', "agama.auto=\"$path\"");
```

Comment:
I guess if EXTRABOOTPARAMS is already set, we will overwrite or perhaps is running before, worth a comment about it if this functionality is limited at the moment, wdyt?

Reply:
seems fine :525 handle those, so should be ok.